interface/forms/requisition/table.sql

```diff
@@ -0,0 +1,11 @@
+--
+-- Table structure for table `requisition`
+--
+
+CREATE TABLE `requisition` (
+  `id` bigint(19) NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  `req_id` varchar(90) NOT NULL,
+  `pid` int(5) NOT NULL,
```

Comment:
Shouldn't the size of this line up with the use of pid in other tables (it seem too small)

Reply:
sized it up!